sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -47,6 +47,16 @@ class FileBrowserElement extends Polymer.Element {
   public currentPath: string;
 
   /**
+   * The type of FileManager we want to use for this file-browser.
```

Comment:
Can we document the different values for the attribute, or point to the factory file maybe?